graphql/resolvers/articleResolver.js

```diff
@@ -286,9 +288,19 @@ module.exports = {
       newArticle.isNew = true
       withUser.articles.push(newArticle)
 
-      //Save the three objects
       await Promise.all([newArticle.save(), withUser.save()])
 
+      // Maintain links
+      await Corpus.updateMany(
+        { 'articles.article': article._id },
+        { $push: { articles: { article: newArticle._id } } }
+      )
+
+      await Workspace.updateMany(
+        { articles: article },
+        { $push: { articles: [newArticle._id] } }
+      )
```

Comment:
on a même pas à préciser "$in" ?

Reply:
Oui ça m'a surpris, ça revient au même que : 

```js
await Workspace.updateMany(
  { articles: { $in: [article] } },
  { $push: { articles: [newArticle._id] } }
)
```